#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_fix_pkfix_invocation.dpatch  <preining@logic.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix invocation of pkfix to get rid of magic

@DPATCH@
 build/source/texk/texlive/pkfix |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

Index: texlive-bin-2006.svn3816/build/source/texk/texlive/pkfix
===================================================================
--- texlive-bin-2006.svn3816.orig/build/source/texk/texlive/pkfix	2006-01-17 22:41:51.000000000 +0100
+++ texlive-bin-2006.svn3816/build/source/texk/texlive/pkfix	2007-01-26 03:56:42.000000000 +0100
@@ -1,7 +1,4 @@
-eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}' && eval 'exec perl -S $0 $argv:q'
-  if 0;
-use strict;
-$^W=1; # turn warning on
+#!/usr/bin/perl -w
 #
 # pkfix.pl
 #
